frontend/src/components/Teams/Team/ListMembers/ListMembers.tsx

```diff
@@ -25,7 +25,7 @@ const ListMembers = ({ isOpen, setIsOpen, isTeamPage }: Props) => {
 
   const router = useRouter();
```

Comment:
I know you didn't change this line but you can do this:

```suggestion
 const {
    query: { teamId },
  } = useRouter();
```